docs/quickstart.rst

```diff
@@ -125,10 +125,10 @@ If you want more control on your data, you can specify additional validators: ::
 If you just need to validate date and not interested in result, you can use
 :func:`~Type.validate()` method: ::
 
-    print UserType.load({"email": "wasa"})
```

Comment:
Please change commit message to something like "Fix validate example in quickstart documentation"